meeteval/io/keyed_text.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+from typing import List
+
+from meeteval.io.base import BaseLine, Base
+
+
+@dataclass(frozen=True)
+class KeyedTextLine(BaseLine):
+    filename: str
+    transcript: str
+
+    @classmethod
+    def parse(cls, line: str) -> 'KeyedTextLine':
+        filename, transcript = line.strip().split(maxsplit=1)
```

Comment:
Can you write a test for this? The `transcript` can be empty and the whitespace can be missing.